repo/packages/M/marathon-lb/19/marathon.json.mustache

```diff
@@ -9,7 +9,23 @@
     "docker": {
       "image": "{{resource.assets.container.docker.marathon-lb-docker}}",
       "network": "HOST",
-      "privileged": true
+      "privileged": true,
+      "parameters": [
+      {{#parameters}}
+        {
+          "key": "{{key}}",
+          "value": "{{value}}"
+        },
+      {{/parameters}}
+        {
+          "key": "created_by",
```

Comment:
Is `--created_by=marathon` a valid flag to pass to `docker run`? Or am I misunderstanding how the `parameters` function?